src/audio/kpb.c

```diff
@@ -58,6 +58,9 @@ DECLARE_TR_CTX(kpb_tr, SOF_UUID(kpb_uuid), LOG_LEVEL_INFO);
 DECLARE_SOF_UUID("kpb-task", kpb_task_uuid, 0xe50057a5, 0x8b27, 0x4db4,
 		 0xbd, 0x79, 0x9a, 0x63, 0x9c, 0xee, 0x5f, 0x50);
 
+#define BYTES_TO_S16_SAMPLES	1
```

Comment:
maybe make this a macro so its not accidentally misused as a multiplication?

Reply:
Yep. There's related macros in kpb.h, I'll move it there.

Reply:
The KPB_BYTES_TO_Sxx_SAMPLES() macros are now in kpb.h.